source/sgp_mode/Tasks.h

```diff
@@ -130,14 +131,15 @@ class TaskSet {
 // These are checked top-to-bottom and the reward is given for the first one
 // that matches
 TaskSet DefaultTasks{
-    {"NOT", InputTask{1, [](auto &x) { return ~x[0]; }, 1.0}, false},
```

Comment:
We should figure out how to have both of these sets of tasks exist in the code without messing each other up. Perhaps this set should be renamed LogicTasks and then the square task should be in a SquareTasks TaskSet and which is used is set by a configuration option?